### ISSUE TITLE
use semantic segmentation for inference on a new dataset without labels

### ISSUE THREAD

**Issue by shayannikoohemat** [AnTao97/dgcnn.pytorch#51]
How can we do the inference for semantic segmentation task on a new dataset without annotation?

Does the same `test()` function in `main_semseg.py` with `eval=True` is enough for inference? Because the evaluation needs the label anyway.

**Comment by antao97** [comment]
I think you can remove the prediction evaluation part of the code. Just only output the prediction results.

**Comment by shayannikoohemat** [comment]
Thank you Antao. Can you be more specific? Do you mean in `def test()`  remove this section `seg_pred = model(data)` and lines related to `pred` ?
But before that when we read the data here `test_loader = DataLoader(S3DIS(partition='test', ...`, it expects labels, should I change the code to skip the label as input? Then where the inferred labels are stored?

**Comment by antao97** [comment]
`seg_pred = model(data)` is the prediction results, not dataset labels.

I mean you can skip all label-related parts.
